alacritty/src/event.rs

```diff
@@ -363,12 +371,19 @@ impl<N: Notify + OnResize> Processor<N> {
                         return;
                     }
                 },
-                // Buffer events
-                _ => {
+                // Remap DPR change event to remove lifetime
+                GlutinEvent::WindowEvent {
+                    event: WindowEvent::ScaleFactorChanged { scale_factor, .. },
+                    ..
+                } => {
                     *control_flow = ControlFlow::Poll;
-                    if !Self::skip_event(&event) {
-                        event_queue.push(event);
-                    }
+                    let event = GlutinEvent::UserEvent(Event::ScaleFactorChanged(scale_factor));
+                    event_queue.push(event);
+                    return;
+                },
+                event => unsafe {
+                    *control_flow = ControlFlow::Poll;
+                    event_queue.push(mem::transmute(event));
```

Comment:
Could you add an explanation why it's safe to do so ;- ) ? ( I mean comment before transmute)